pylal/SimInspiralUtils.py

```diff
@@ -42,10 +42,7 @@ class ExtractSimInspiralTableLIGOLWContentHandler(ligolw.PartialLIGOLWContentHan
   """
   def __init__(self,document):
     def filterfunc(name,attrs):
-      if name==ligolw.Table.tagName and attrs.has_key('Name'):
-        return 0==table.CompareTableNames(attrs.get('Name'), lsctables.SimInspiralTable.tableName)
-      else:
-        return False
+      return ligolw.PartialLIGOLWContentHandler(document, lambda name, attrs: name == lsctables.SimInspiralTable.tableName)
```

Comment:
I think you just want

```python
         def filterfunc(name, attrs):
            return lsctables.SimInspiralTable.CheckProperties(name, attrs)
```

Reply:
Thanks, @duncanmmacleod!  I followed your advice and tested again.  It all works.

@a-r-williamson, ready to merge.